docs/api.md

```diff
@@ -38,6 +38,11 @@
     :docstring:
     :members: headers cookies params request get head options post put patch delete build_request send close
 
+::: httpx.AsyncClient
```

Comment:
Let's put this in a dedicated section maybe to make it more easily discoverable in the navigation?

```suggestion
## `AsyncClient`

::: httpx.AsyncClient
```

![Screenshot 2020-03-28 at 15 44 38](https://user-images.githubusercontent.com/15911462/77825671-12127580-710b-11ea-88ea-2903f5686476.png)